netcore/NetCore.Session.IntegrationTests/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "Apache.Geode.Session.IntegrationTests": {
+      "commandName": "Project",
+      "nativeDebugging": true
+    }
+  }
+}
```

Comment:
Missing newline at EOF.  Where there's one of these, there will probably be others.

Reply:
Done